sliceutil/sets.go

```diff
@@ -41,3 +41,19 @@ func Complement(a, b []int64) (aOnly, bOnly []int64) {
 
 	return aOnly, bOnly
 }
+
+// Intersection returns the elements common to both a and b
+func Intersection(a, b []int64) []int64 {
+	inter := []int64{}
+	hash := make(map[int64]bool)
```

Comment:
`hash := make(map[int64]bool, len(a))` might be more performant with larger slices, no? 🤔